particles/Tutorial/Javascript/Demo/source/RandomComputer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/* global defineParticle */
+
+defineParticle(({DomParticle, html}) => {
+
+  return class extends DomParticle {
+
+    update({gameState}) {
```

Comment:
I still don't see the `Game` telling players when it is their turn to make the move.

Without this done, it will be hard to have both randomised "first mover advantage" and an indication in the UI whether it is your turn or whether you are waiting for the other player to make the move.

Reply:
I also encourage making some randomised wait time in the random computer, so there's more of a feeling that you're playing against something that thinks :)

Reply:
So the time is effectively random because it's based on when the human clicks.

Reply:
It appears like we didn't understand each other, so let me try again:

Players should know when it is their turn to make the move. This could be for example by also reading the "currentPlayer" field of the GameState. Computer should only generate a move if it is its turn. Human should only be allowed to make a move if it is its turn. When it is computer's turn, Board should be disabled, there should be UI that makes it clear that we're waiting for another player. When it becomes computer's turn, we could make a move after 300ms + Random() * 200ms, so that there is this feeling of playing against something thinking.

In the context of the above, the UX of TTT would be:
1) I (human) make a move
2) Board gets disabled, mouse cursor could change from cursor: pointer; to cursor: not-allowed; or the spinner appears, something like that.
3) After some time computer makes a move.
4) Boards allows me again to make a move.

In the light of the above I don't understand "time is effectively random because it's based on when the human clicks."